### PR TITLE
feat: add student posts API

### LINKED ISSUE #300: Add student posts (platoon leader, etc)

### PR DESCRIPTION
**Pull request by vladisa88**
Added API for Student Posts. It has been done like Teacher Posts. Resolves #300

### CODE REVIEW

**Review by gakhromov — CHANGES_REQUESTED** [review]
Remove the unnecessary Pipfile. Otherwise, everything LGTM

**Review by gakhromov — APPROVED** [review]
LGTM

File: back-end/src/common/views/populate.py [hunk 1]
```diff
@@ -50,30 +50,14 @@
...
-    create_uniforms,
-)
+    create_faculties, create_programs, create_milfaculties,
+    create_milspecialties, create_milgroups, create_ranks, create_posts,
+    create_passports, create_recruitments_offices, create_university_infos,
+    create_students, create_teachers, create_absences, create_punishments,
+    create_encouragements, create_achievement_types, create_achievements,
+    create_subjects as create_lms_subjects, create_rooms, create_lessons,
+    create_absence_restriction_time, create_marks, create_uniforms,
+    create_student_posts)
```
Comment (TmLev) [inline back-end/src/common/views/populate.py:60]: Add a trailing comma and preserve one-per-line import with formatter.

File: back-end/src/lms/models/students.py [hunk 2]
```diff
@@ -18,15 +18,28 @@
 )
 
 
+class StudentPost(models.Model):
+    title = models.CharField(primary_key=True,
```
Comment (TmLev) [inline back-end/src/lms/models/students.py:22]: `primary_key=True` is bad. @gakhromov

File: back-end/src/lms/views/populate.py [hunk 3]
```diff
@@ -16,11 +16,8 @@
     Program,
     UniversityInfo,
 )
-from lms.models.students import (
-    Student,
-    Passport,
-    RecruitmentOffice,
-)
+from lms.models.students import (Student, Passport, RecruitmentOffice,
+                                 StudentPost)
```
Comment (TmLev) [inline back-end/src/lms/views/populate.py:20]: Trailing comma and one-per-line import.

**Review by TmLev — CHANGES_REQUESTED** [review]
.

File: back-end/src/lms/models/students.py (hunk 2 above)
Comment (gakhromov) [reply back-end/src/lms/models/students.py]: Let's leave it as is for now. We will return to this issue later

**Review by gakhromov — APPROVED** [review]
LGTM

**Review by TmLev — APPROVED** [review]
LGTM.

**Pull request merged**